profile.d/meson.build

```diff
@@ -0,0 +1,10 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+install_data('70-systemd-shell-extra.sh', install_dir : shellprofiledir.startswith('/usr/') ? shellprofiledir : libexecdir / 'profile.d')
+
+if not shellprofiledir.startswith('/usr/')
```

Comment:
Why not to use `LINK_SHELL_EXTRA_DROPIN` ? `shellprofiledir` may be `"no"`.

Reply:
hmm, i copied this from the sshd_config/ssh_config drop-ins. i guess i should update that too then.